src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent-docker.mdx

```diff
@@ -53,23 +57,27 @@ With just a few additions your existing Dockerfile can be used with our Node.js
 
 ## Other configuration options [#configure]
 
-<Callout variant="caution">
-  Don't include your license key in your Dockerfile or Docker image. For more information, see [our documentation on license key security](/docs/accounts/install-new-relic/account-setup/license-key#license-key-security).
+<Callout variant="important">
```

Comment:
Can this be part of a separate h2 section at the start of the doc about versioning, instead of a callout? What's the benefit of it being a callout?

Reply:
This was a callout early on in this document probably because it's a warning. I'm in favor of hoisting it higher.